agot-bg-game-server/src/common/ingame-game-state/game-data-structure/createGame.ts

```diff
@@ -272,6 +289,11 @@ export default function createGame(housesToCreate: string[]): Game {
             const unitType = unitTypes.get(unitData.unitType);
             const quantity = unitData.quantity;
 
+            // Check if the game setup removed units off this region
+            if (gameSetup.removedUnits && gameSetup.removedUnits.includes(region.id)) {
+                return;
+            }
+
```

Comment:
It's not the most ideal way of handling this, but it works, and I don't think we will need to refactor this any time soon.